bool_value: True
int_value: 5
double_value: 3.14
string_value: "Hello Universe"
bool_array: [True, False, True]
int_array: [10, 11, 12, 13]
int_array_2: [10.1, 11.2, 12.3, 13.4]
double_array: [7.5, 400.4] #
double_array_2: [7, 400] #
string_array: ['Nice', 'more', 'params']
bytes_array: [0x01, 0xF1, 0xA2]
nested_param:
  another_int: 7
  another_int2: 7.0
  nested_param:
    another_int: 7
    another_int2: 7.0